resources/views/home/index.blade.php

```diff
@@ -0,0 +1,20 @@
+@extends('layouts.master')
```

Comment:
You can safely remove this file

Reply:
If you want you can add an About section on routes for this kind of information. Maybe authors would be included there!